src/gui/owncloudgui.cpp

```diff
@@ -454,51 +475,65 @@ void ownCloudGui::setupContextMenu()
         return;
     }
 
-// Enables workarounds for bugs introduced in Qt 5.5.0
-// In particular QTBUG-47863 #3672 (tray menu fails to update and
-// becomes unresponsive) and QTBUG-48068 #3722 (click signal is
-// emitted several times)
-// The Qt version check intentionally uses 5.0.0 (where platformMenu()
-// was introduced) instead of 5.5.0 to avoid issues where the Qt
-// version used to build is different from the one used at runtime.
-// If we build with 5.6.1 or newer, we can skip this because the
-// bugs should be fixed there.
-#ifdef Q_OS_LINUX
-#if (QT_VERSION >= QT_VERSION_CHECK(5, 0, 0)) && (QT_VERSION < QT_VERSION_CHECK(5, 6, 0))
```

Comment:
I think 2.4 still support old Qt version.  Maybe not wise to get rid of this... or?

Reply:
Linux builds with 2.4.x use Qt-5.6.2 everywhere.
Not sure about win or osx, though.

Reply:
the official build, yes, but the distribution provided package or community packages, maybe not.
Although it is true that it is unlikely that a distribution would package the client while having Qt 5.5.